src/formatDate.js

```diff
@@ -51,6 +51,59 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const formatingDate = date.split(fromFormat[3]);
+  const newDate = [];
+  let year = '';
+  let month = '';
+  let day = '';
+
+  for (let i = 0; i < 3; i++) {
+    switch (fromFormat[i]) {
+      case 'YYYY':
+        year = formatingDate[i];
+        break;
+
+      case 'YY':
+        year = formatingDate[i];
+
+        if (year > 23) {
+          year = 19 + year;
+        } else {
+          year = 20 + year;
+        }
+        break;
+
+      case 'MM':
+        month = formatingDate[i];
+        break;
+
+      case 'DD':
+        day = formatingDate[i];
+        break;
+    }
```

Comment:
Default cases are missed in both loops. Add them, please. You always should add a default case to the switch/case statement to prevent bags